tests/dom_tests.js

```diff
@@ -174,6 +174,11 @@ QUnit.test("matches", function(assert) {
   assert.ok(document.querySelector('#baz').matches('.beta'));
 });
 
+QUnit.test("closest", function(assert) {
+  assert.equal(document.querySelector('#bar').closest('#foo'), document.querySelector('#foo'));
+  assert.equal(document.querySelector('#baz').closest('#foo'), document.querySelector('#foo'));
+});
```

Comment:
Aha, neat. I've added 2 more tests now, will that suffice? I tried to test it in IE 8 in BrowserStack but the tests wont run (got a blank page) even without my modifications. Work fine in IE 9 and up though. Can you check if you get the same problem?

If I get time sometime I might create a CI PR!